src/tools/rbd/action/Bench.cc

```diff
@@ -196,6 +196,10 @@ int do_bench(librbd::Image& image, io_type_t io_type,
     return -EINVAL;
   }
 
+  if (io_size > 4294967295) {
+    std::cerr << "rbd: io-size should be less than 4G" << std::endl;
```

Comment:
it should not be <= because 4G is overflowing. 4G=4294967296  and after overflow value will become 0